test/helpers/create-https-test-server.ts

```diff
@@ -0,0 +1,72 @@
+import https = require('https');
+import net = require('net');
+import express = require('express');
+import pify = require('pify');
+import pem = require('pem');
+
+export type HttpsServerOptions = {
+	commonName?: string;
+	days?: number;
+};
+
+export interface ExtendedHttpsTestServer extends express.Express {
+	https: https.Server;
+	caKey: Buffer;
+	caCert: Buffer;
+	url: string;
+	port: number;
+	close: () => Promise<any>;
+}
+
+const createHttpsTestServer = async (options: HttpsServerOptions = {}): Promise<ExtendedHttpsTestServer> => {
+	const createCSR = pify(pem.createCSR);
```

Comment:
```suggestion
	const createCertificateSingingRequest = pify(pem.createCSR);
```

Reply:
I prefer to leave it as it is because is a mirror of the name of the original function.

BTW: I see that this conversation is marked as resolved, so is it fine?